configs/samples/res_usbradio.conf.sample

```diff
@@ -0,0 +1,7 @@
+[general]
+;usb_devices = 1209:7388	;comma delimited list of usb
+							;descriptors to allow.
+							;format vvvv:pppp in hexadecimal
+							;vvvv=vendor id, pppp=product id
+							;
+							;1209:7388 = AIOC (all in one cable)
```

Comment:
To help those viewing/editing this file I'd ask that we replace the tabs with spaces?